code/_onclick/click.dm

```diff
@@ -129,7 +129,7 @@
 
 	// Allows you to click on a box's contents, if that box is on the ground, but no deeper than that
 	if(isturf(A) || isturf(A.loc) || (A.loc && isturf(A.loc.loc)))
-		if(A.Adjacent(src)) // see adjacent.dm
+		if(A.Adjacent(src) || W.reach >= get_dist(src, A)) // see adjacent.dm

```

Comment:
It would, but this if is an OR (```||```)
```
var/A = FALSE
var/B = TRUE
if (A || B)
    world << "wew"
```
this, not surprisingly, produces "wew"